### DISCUSSION TITLE
编译的swoole.so怎么这么大？

### DISCUSSION THREAD

**Discussion by w22gb8** [swoole/swoole-src#4994] [General]
arm64单独编译的swoole.so扩展51M ，而swoole-cli包含php核心和其它扩展才31M，啥情况？

**Comment by NathanFreeman** [comment]
官网下载的swoole-cli有通过strip这个函数做处理，占用大小会小一点。

**Reply by w22gb8** [reply]
> 官网下载的swoole-cli有通过strip这个函数做处理，占用大小会小一点。

谢谢回复，通过strip -s swoole.so 减小到2.6M。
